From: Václav Ovsík <vaclav.ovsik@gmail.com>
Subject: [PATCH] fixes/stdarg-va_copy

This patch fixes compiler warning:
ustr-compiler.h:19:7: warning: "__va_copy" is not defined [-Wundef]

Signed-off-by: Václav Ovsík <vaclav.ovsik@gmail.com>

---
 ustr-compiler.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ustr-compiler.h b/ustr-compiler.h
index 9e71276..642e6e8 100644
--- a/ustr-compiler.h
+++ b/ustr-compiler.h
@@ -16,7 +16,7 @@
 #ifdef va_copy
 # define USTR_CONF_HAVE_VA_COPY 1
 # define USTR__VA_COPY(x, y)   va_copy(x, y)
-#elif __va_copy
+#elif defined(__va_copy)
 # define USTR_CONF_HAVE_VA_COPY 1
 # define USTR__VA_COPY(x, y) __va_copy(x, y)
 #else
-- 
tg: (b148fb7..) fixes/stdarg-va_copy (depends on: upstream)
